### ISSUE TITLE
Support Gradle 8.0

### ISSUE THREAD

**Issue by Goooler** [asciidoctor/asciidoctor-gradle-plugin#664]
Links #655.

**Comment by jmewes** [comment]
A build using Gradle 7.3.3 creates the following warning message from `asciidoctor` task for us:

```
$ ./gradlew clean build --warning-mode all 

> Task :asciidoctor
The JavaExecHandleBuilder.setMain(String) method has been deprecated. This is scheduled to be removed in Gradle 8.0. Please use the mainClass property instead. Consult the upgrading guide for further information: https://docs.gradle.org/7.3.3/userguide/upgrading_version_7.html#java_exec_properties

BUILD SUCCESSFUL in 22s
10 actionable tasks: 10 executed
```

I suppose that this is coming from the following line of code:

https://github.com/asciidoctor/asciidoctor-gradle-plugin/blob/996df6a/jvm/src/main/groovy/org/asciidoctor/gradle/jvm/AbstractAsciidoctorTask.groovy#L582

**Also see**

- https://docs.gradle.org/7.0/javadoc/org/gradle/process/JavaExecSpec.html
- https://docs.gradle.org/8.0/javadoc/org/gradle/process/JavaExecSpec.html
- https://youtrack.jetbrains.com/issue/IDEA-279280/gradle-The-JavaExec.main-property-has-been-deprecated

**Comment by ysb33r** [comment]
This should be fixed when 4.0.0-alpha.2 comes out
